std/zig/ast.zig

```diff
@@ -1708,15 +1710,15 @@ pub const Node = struct {
             i -= 1;
 
             switch (self.op) {
-                @TagType(Op).Call => |*call_info| {
+                .Call => |*call_info| {
                     if (i < call_info.params.len) return call_info.params.at(i).*;
                     i -= call_info.params.len;
                 },
-                Op.ArrayAccess => |index_expr| {
+                .ArrayAccess => |index_expr| {
```

Comment:
I don't know if there's a difference between `@TagType(Op)` and `Op` but this compiles just fine.